src/app/complex/complex-details/complex-participants/complex-participants.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="row" *ngIf="participantsWithSubcomponents">
-  <h2 class="float-left">Participants</h2>
+  <h2 class="float-left">Components</h2>
```

Comment:
Since we change this, we should also change it in `go-to.component.html`, line 9, from Participants to Components.

Reply:
Then, in `complex-filter.component.html`, should it be `Interactor Type` or `Component Type`? Can you check that with Sucharita?

Reply:
As Sucharita said, let's rename this as Component Type.